src/sentry/static/sentry/app/views/performance/realUserMonitoring/index.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import {Location} from 'history';
+import {browserHistory, WithRouterProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import Alert from 'app/components/alert';
+import Feature from 'app/components/acl/feature';
+import Redirect from 'app/utils/redirect';
+import LightWeightNoProjectMessage from 'app/components/lightWeightNoProjectMessage';
+import GlobalSelectionHeader from 'app/components/organizations/globalSelectionHeader';
+import SentryDocumentTitle from 'app/components/sentryDocumentTitle';
+import {t} from 'app/locale';
+import {PageContent} from 'app/styles/organization';
+import {Organization, Project, GlobalSelection} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import {isAggregateField} from 'app/utils/discover/fields';
+import {decodeScalar} from 'app/utils/queryString';
+import {tokenizeSearch, stringifyQueryObject} from 'app/utils/tokenizeSearch';
+import withGlobalSelection from 'app/utils/withGlobalSelection';
+import withOrganization from 'app/utils/withOrganization';
+import withProjects from 'app/utils/withProjects';
+
+import RumContent from './content';
+import {getTransactionName} from '../utils';
+import {transactionSummaryRouteWithQuery} from '../transactionSummary/utils';
+
+type Props = {
+  location: Location;
+  organization: Organization;
+  projects: Project[];
+  selection: GlobalSelection;
+} & Pick<WithRouterProps, 'router'>;
+
+type State = {
+  eventView: EventView | undefined;
+};
+
+class RealUserMonitoring extends React.Component<Props> {
+  state: State = {
+    eventView: generateRumEventView(
+      this.props.location,
+      getTransactionName(this.props.location)
+    ),
+  };
+
+  static getDerivedStateFromProps(nextProps: Props, prevState: State): State {
+    return {
+      ...prevState,
+      eventView: generateRumEventView(
+        nextProps.location,
+        getTransactionName(nextProps.location)
+      ),
+    };
+  }
+
+  getDocumentTitle(): string {
+    const name = getTransactionName(this.props.location);
+
+    const hasTransactionName = typeof name === 'string' && String(name).trim().length > 0;
+
+    if (hasTransactionName) {
+      return [String(name).trim(), t('RUM')].join(' - ');
+    }
+
+    return [t('Summary'), t('RUM')].join(' - ');
```

Comment:
```suggestion
      return [String(name).trim(), t('RUM')].join(' \u2014 ');
    }

    return [t('Summary'), t('RUM')].join(' \u2014 ');
```

Proper mdashes look a smidge better.

Reply:
Addressed in 33a7282